pkg/koordlet/runtimehooks/hooks/gpu/gpu.go

```diff
@@ -70,5 +71,31 @@ func (p *gpuPlugin) InjectContainerGPUEnv(proto protocol.HooksProtocol) error {
 		containerCtx.Response.AddContainerEnvs = make(map[string]string)
 	}
 	containerCtx.Response.AddContainerEnvs[GpuAllocEnv] = strings.Join(gpuIDs, ",")
+	if containerReq.PodLabels[ext.LabelGPUIsolationProvider] == string(ext.GPUIsolationProviderHAMICore) {
+		gpuResources := devices[0].Resources
```

Comment:
Shall we ensure the target GPU info is on the first device?

Reply:
currently, we assure all gpu device has the same allocation result